### PR TITLE
examples: updates to the latest SDKs

### PR DESCRIPTION
**Pull request by codefromthecrypt**
This updates our examples to the latest SDKs as particularly this can help reveal issues around fan-out stats.

### CODE REVIEW

**Review by mathetake — APPROVED** [review]

**Comment by codefromthecrypt** [comment]
ok so the hang on the other PR is about latest zig, not the changes

**Comment by mathetake** [comment]
why did you land this on main if it's hanging indefinitely?

**Comment by mathetake** [comment]
oh ok reverted the change in intergration.yml :D

**Comment by codefromthecrypt** [comment]
I'm not positive the hang is completely gone, but it didn't happen last time. we'll see

**Comment by codefromthecrypt** [comment]
the hang wasn't gone, just got a lucky run. looking into it on #1168 since it was the same crash there

File: .github/workflows/integration.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ defaults:
     shell: bash
 
 env:  # Update this prior to requiring a higher minor version in go.mod
-  GO_VERSION: "1.19"  # 1.xx == latest patch of 1.xx
+  GO_VERSION: "1.20"  # 1.xx == latest patch of 1.xx
```
[the remark this answers is not in the document]
Comment (codefromthecrypt) [reply .github/workflows/integration.yaml:21]: actually go 1.20 is the only version that passes regularly all tests except one in zig. I've not identified that cause yet.

**Pull request merged**